jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -144,6 +148,10 @@ def run(params) {
                             res_sync_mu_repos = sh(script: "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd '${env.exports} cd /root/spacewalk/testsuite; rake cucumber:build_validation_wait_for_custom_reposync'")
                             echo "Custom channels and MU repositories synchronization status code: ${res_sync_mu_repos}"
                             sh "exit \$(( ${res_mu_repos}|${res_sync_mu_repos} ))"
+                            // Add persistent monitoring state
+                            // Explanation : monitoring is using the same MU than one of the sles minion, we can't add twice the MUs
+                            // We are using a persistent file to show monitoring MU has been added
+                            sh "touch ${env.resultdir}/sumaform/monitoring"
```

Comment:
If we want to use lock files, better if we do inside a folder with a self explanatory name, and a file name that ends in .lock to make it clear.